src/containerConfig.ts

```diff
@@ -1,18 +1,25 @@
+import * as fs from 'fs';
```

Comment:
you can import only promises
```suggestion
import { promises } from 'fs';
```

Reply:
done.